proto/v1beta1/spdx.proto

```diff
@@ -0,0 +1,391 @@
+// Copyright 2021 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grafeas.v1beta1.spdx;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/grafeas/grafeas/proto/v1beta1/spdx_go_proto";
+option java_multiple_files = true;
+option java_package = "io.grafeas.v1beta1.spdx";
+option objc_class_prefix = "GRA";
+
+// DocumentNote represents an SPDX Document Creation Infromation section:
+// https://spdx.github.io/spdx-spec/2-document-creation-information/
+message DocumentNote {
```

Comment:
Is there a particular reason these were merged into one file instead of the split? In the previous review of the SPDX elements, I asked for the split -- perhaps there's a drawback to having it?

Reply:
The only reason for having the SPDX protos in a single file was that they are all related, similarly we have a single file for in-toto.

Reply:
I see, thanks. I'm fine with either approach.